plugins/thunderbolt/README.md

```diff
@@ -44,6 +44,12 @@ The retimer index is oriented around the physical connection within
 the machine.  It is important as multiple controllers may otherwise
 identify identically.
 
+Update Behavior
+---------------
+
+The firmware is written to the device at runtime. Once complete the controller
+may reboot which may cause all connected USB devices to be reenumerated.
```

Comment:
This probably needs some elaboration since flashing WD19 and config option change it.

Reply:
Got anything more please? You understand this much better than me.